dev-infrastructure/templates/dev-acr.bicep

```diff
@@ -9,6 +9,23 @@ param location string = resourceGroup().location
 @description('Service tier of the Azure Container Registry.')
 param acrSku string
 
+@description('KeyVault secret name with the password used to log into quay.')
+#disable-next-line secure-secrets-in-params
+param passwordSecretIdentifier string = 'quay-password'
+
+@description('KeyVault secret name with the username used to log into quay.')
+#disable-next-line secure-secrets-in-params
+param usernameSecretIdentifier string = 'quay-username'
+
+@description('List of quay repositories to cache in the Azure Container Registry.')
+param quayRepositoriesToCache array = []
+
+@description('Name of the global key vault.')
+param globalKeyVaultName string = ''
```

Comment:
can we call this just key vault. gives us some flexibility about what KV will store these creds